plugins/woocommerce/templates/emails/plain/email-order-details.php

```diff
@@ -43,7 +43,7 @@
 }
 
 if ( $order->get_customer_note() ) {
-	echo esc_html__( 'Note:', 'woocommerce' ) . "\t " . wp_kses( wptexturize( $order->get_customer_note() ), array() ) . "\n";
+	echo esc_html__( 'Note:', 'woocommerce' ) . "\t " . wp_kses( wptexturize( $order->get_customer_note() ), array( 'br' => array() ) ) . "\n";
```

Comment:
This one is in plain text emails, so I think this could stay as it was before (an empty array) so we don't output any HTML in plain text email (e.g. if the customer manually saved `<br>` in the note).